tests/jlm/llvm/frontend/llvm/LoadTests.cpp

```diff
@@ -25,7 +25,8 @@ LoadConversion()
   std::unique_ptr<Module> llvmModule(new Module("module", context));
 
   auto int64Type = Type::getInt64Ty(context);
-  auto pointerType = Type::getInt64PtrTy(context);
```

Comment:
The Int64PtrTy seems to be deprecated.

Reply:
The same here. It is still the one with the type argument.